src/main/java/org/kiwitcms/java/model/TcmsTestCaseId.java

```diff
@@ -0,0 +1,13 @@
+package org.kiwitcms.java.model;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface TcmsTestCaseId
+{
+	int value();
+}
```

Comment:
What is the purpose of this class?  Is  this where the new annotation is defined?  If yes, I think `model/` isn't the appropriate place for it. Maybe create a new directory like `annotations/` ? WDYT ?

Reply:
Or maybe add this class into `TestMethod.java` ? Not sure what makes sense here.